engine/controllers/server.h

```diff
@@ -25,7 +25,8 @@ namespace inferences {
 class server : public drogon::HttpController<server, false>,
                public BaseModel,
                public BaseChatCompletion,
-               public BaseEmbedding {
+               public BaseEmbedding,
+               public BasePythonModel {
```

Comment:
Maybe I'm missing something, I'm not sure why do we need to create interface like BaseModel, BasePythonModel for this server controller?